scripts/draw_perfboard.py

```
@@ -0,0 +1,288 @@
+# ---------------- jumper wires ----------------
+# +5V feeds (header +5V pins -> +5V rail). +5V pins at cols 8 (J5), 17 (J4), 20 (J3), 23 (J6).
+for c in (8, 17, 20, 23):
+    wire((c, 'A'), (c, 'P'), RED)
+# GND feeds (header GND pins -> GND rail). GND pins at cols 1, 4, 7 (left)  and 18, 21, 24 (right).
+for c in (1, 4, 7, 18, 21, 24):
+    wire((c, 'E'), (c, 'G'), BLK)
+# Q1 emitter (col 13) -> GND rail
+wire((13, 'E'), (13, 'G'), BLK)
+# XBUS_TX jumper: R2 right (col 11) -> J3.sig (col 19)
+wire((11, 'C'), (19, 'C'), BLU)
+join_dot(11, 'C', BLU); join_dot(19, 'C', BLU)
```

Ruff (0.15.17)

[error] Multiple statements on one line (semicolon)

(E702)